src/vss_tools/vspec/vssexporters/vss2samm/helpers/namespaces.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from rdflib import URIRef
+from vss_tools import log
+from .. config import config as cnfg
+
+
+def get_vspec_uri(node_name: str):
+    return URIRef(f'{samm_output_namespace}{node_name}')
```

Comment:
why is the namespace not passed as an arg? I would try to avoid  global vars whenever possible